subsys/bluetooth/audio/shell/audio.h

```diff
@@ -375,14 +375,10 @@ static inline void print_codec_meta_program_info(const struct shell *sh, size_t
 }
 
 static inline void print_codec_meta_language(const struct shell *sh, size_t indent,
-					     uint32_t stream_lang)
+					     const uint8_t lang[BT_AUDIO_LANG_SIZE])
 {
-	uint8_t lang_array[3];
-
-	sys_put_be24(stream_lang, lang_array);
-
-	shell_print(sh, "%*sLanguage: %c%c%c", indent, "", (char)lang_array[0], (char)lang_array[1],
-		    (char)lang_array[2]);
+	shell_print(sh, "%*sLanguage: %c%c%c", indent, "", (char)lang[0], (char)lang[1],
```

Comment:
Could `%.*s` (with `3, lang`) be used instead of %c%c%c ?

Reply:
Does that work with non-terminated strings?

Reply:
It should, afaik - as it then clearly states that N chars are used